### PR TITLE
Fix empty lines rules

### PR DESCRIPTION
**Pull request by vankop**
> Which issue, if any, is this issue related to?

related to https://github.com/stylelint/stylelint/issues/4232 . Now there is no extra warnings.

fix:
- `function-max-empty-lines`
- `selector-max-empty-lines`
- `value-list-max-empty-lines`

> Is there anything in the PR that needs further explanation?

Only for `value-list-max-empty-lines` was test case with comments with exited max empty lines length, so only in the end of this work I realized that comments inside rule range are skipped =(
Also there is no info in README about it

If this behavior must be restored I can rework

### CODE REVIEW

File: lib/rules/function-max-empty-lines/index.js [hunk 1]
```diff
@@ -27,66 +30,82 @@ const rule = function(max, options, context) {
...
+          splittedValue.push([
+            stringValue.slice(sourceIndexStart, node.sourceIndex),
+            newNodeString
+          ]);
+          sourceIndexStart = node.sourceIndex + stringifiedNode.length;
+        } else {
+          report({
+            message: messages.expected(max),
+            node: decl,
+            index: placeIndexOnValueStart(decl) + node.sourceIndex,
```
Comment (vankop) [inline lib/rules/function-max-empty-lines/index.js]: so for `function-max-empty-lines` column will be set to start of the function
```css
a { transform: translate() }
/*             ↑ */
/*  column will start here */            
```

File: lib/rules/selector-max-empty-lines/index.js [hunk 2]
```diff
@@ -26,61 +25,52 @@ const rule = function(max, options, context) {
...
         }
-      });
+      } else if (
+        violatedLFNewLinesRegex.test(selector) ||
+        violatedCRLFNewLinesRegex.test(selector)
+      ) {
+        report({
+          message: messages.expected(max),
+          node: rule,
+          index: 0,
```
Comment (vankop) [inline lib/rules/selector-max-empty-lines/index.js]: for `selector-max-empty-lines` column set to start of selector

File: lib/rules/value-list-max-empty-lines/index.js [hunk 3]
```diff
@@ -26,64 +25,49 @@ const rule = function(max, options, context) {
...
+          decl.value = newValueString;
         }
+      } else if (
+        violatedLFNewLinesRegex.test(value) ||
+        violatedCRLFNewLinesRegex.test(value)
+      ) {
+        report({
+          message: messages.expected(max),
+          node: decl,
+          index: 0,
```
Comment (vankop) [inline lib/rules/value-list-max-empty-lines/index.js]: for `value-list-max-empty-lines` column set to start of the property

**Comment by hudochenkov** [comment]
@vankop could you help summarize this PR in one sentence for commit message and changelog entry, please? :)

**Comment by vankop** [comment]
Rework `*-max-empty-lines` rules to create only one violation per function, selector, value list

**Comment by jeddy3** [comment]
In Changelog parlance:

- Fixed: `*-max-empty-lines` to only report one violation per function, selector, value list

**Comment by hudochenkov** [comment]
> -   Fixed: `*-max-empty-lines` to only report one violation per function, selector, value list ([#4260](https://github.com/stylelint/stylelint/pull/4260)).

**Pull request merged**